test/legacy_test/test_deformable_conv_op.py

```diff
@@ -155,7 +156,7 @@ def deform_conv2d_wrapper(
 class TestModulatedDeformableConvOp(OpTest):
     def setUp(self):
         self.python_api = deform_conv2d_wrapper
-        self.op_type = "deformable_conv"
+        self.op_type = "deform_conv2d"
```

Comment:
deform_conv2d api 对应的 op_type 是 "deformable_conv"，所以这里不用修改
```suggestion
        self.op_type = "deformable_conv"
```